plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -126,7 +149,32 @@ fu_pxi_device_calculate_checksum (const guint8 *data, gsize sz)
 	return checksum;
 }
 
-/* FIXME: is @port required? */
+static gboolean
+fu_pxi_device_check_support_resume (FuPxiDevice *self,
+				 	FuFirmware *firmware,
+				  	GError **error)
+{
+	g_autoptr(GBytes) fw = NULL;
+	g_autoptr(GPtrArray) chunks = NULL;
+	guint16 checksum_tmp = 0x0;
+	
+	/* get the default image */
+	fw = fu_firmware_get_image_default_bytes (firmware, error);
```

Comment:
you need the `if (fw = NULL) return FALSE` line too.

Reply:
And I need to set the GError ?

Reply:
No, fu_firmware_get_image_default_bytes sets the GError if it returns with NULL.